adminSiteServer/apiRouter.ts

```diff
@@ -2287,67 +2293,75 @@ getRouteNonIdempotentWithRWTransaction(
     }
 )
 
+/**
+ * Handles all four `GdocPublishingAction` cases
+ * - Saving (no action)
+ * - Publishing (index and bake)
+ * - Updating (index and bake (potentially via lightning deploy))
+ * - Unpublishing (remove from index and bake)
+ */
+async function indexAndBakeGdocIfNeccesary(
```

Comment:
Maybe it makes sense to move this one to `gdocsDeploy`, too?

Reply:
Yeahhh I had the same desire but those functions are in the `adminSiteClient` folder, and I didn't want to make a new file just for one function so I kept it close to usage instead.